### DISCUSSION TITLE
Welcome to mappView-File-Explorer Discussions!

### DISCUSSION THREAD

**Discussion by stephan1827** [stephan1827/mappView-File-Explorer#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cardelgil** [comment]
Hello, im programming something for my school in mappview, but i have a problem with the file manager, i was looking your code but i dont know how to integrate my code with your code. The name of my Device is CF and i change it that in your program, but doesnt work. Do i need to change anything else? i hope that you can help me with this and thanks.

**Reply by stephan1827** [reply]
Hi, I can try to help you. Did you follow this guide to import the sample into your project?

https://github.com/br-automation-com/mappView-File-Explorer/blob/master/Logical/mappFileExplorer/HowToImport.pdf

Do you see an error message? Please also send a screenshot of your file device configuration.
